src/components/ApplicationForm/Questionnaire.js

```diff
@@ -188,6 +188,34 @@ export default ({ errors, formInputs, onChange }) => {
             })
           }
         />
+        <Select
+          type="checkbox"
+          label="cmd-f 2022"
+          checked={formInputs.eventsAttended.cmdf2022}
+          onChange={() =>
+            onChange({
+              ...formInputs,
+              eventsAttended: {
+                ...formInputs.eventsAttended,
+                cmdf2022: !formInputs.eventsAttended.cmdf2022,
+              },
+            })
+          }
+        />
+        <Select
+          type="checkbox"
+          label="cmd-f 2023"
```

Comment:
"cmd-f 2023"  -> "nwHacks 2024"

Reply:
Wait why do we need to make this change? The events attended checkboxes update properly: 

![image](https://github.com/nwplus/portal/assets/62491197/30d6f99a-86aa-4100-af62-e116ba0dee29)
![image](https://github.com/nwplus/portal/assets/62491197/5ffa840d-f876-48e5-b7e7-f6a42ac6955b)